axelrod/match.py

```diff
@@ -28,6 +28,7 @@ def __init__(self, players, turns, deterministic_cache=None,
         self.result = []
         self.player1 = players[0]
```

Comment:
I agree. I've just pushed just going with `players`. That's more flexible and doesn't change any actual functionality.